### PR TITLE
CB-17735: Generate event when resize is complete.

### PR DESCRIPTION
**Pull request by kkalvagadda1**
See detailed description in the commit message.

### CODE REVIEW

**Review by RokolyA94 — APPROVED** [review]

File: flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java [hunk 1]
```diff
@@ -117,4 +118,8 @@ public FlowChainLog save(FlowChainLog chainLog) {
         return repository.save(chainLog);
     }
 
+    public boolean isFlowTriggeredByFlowChain(String flowChainType, FlowLogWithoutPayload lastFlowLog) {
```
Comment (bergerdenes) [inline flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java:121]: test?

**Review by bergerdenes — CHANGES_REQUESTED** [review]

**Review by lnardai — APPROVED** [review]
unit test then we are good to go.

File: flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java (hunk 1 above)
Comment (kkalvagadda1) [reply flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowChainLogService.java:121]: Added unit test for the same

File: flow/src/test/java/com/sequenceiq/flow/service/flowlog/FlowChainLogServiceTest.java [hunk 2]
```diff
@@ -131,6 +134,19 @@ public void testFlowChainLogWithFlowChainType() {
         assertEquals("flowChainType", flowChainType);
     }
 
+    @Test
+    public void testIsFlowTriggeredByFlowChain() {
+        FlowLogWithoutPayload flowLog = mock(FlowLogWithoutPayload.class);
+        lenient().when(flowLog.getFlowChainId()).thenReturn("chainId");
```
Comment (bergerdenes) [inline flow/src/test/java/com/sequenceiq/flow/service/flowlog/FlowChainLogServiceTest.java:140]: why lenient? it is a concrete test

**Review by bergerdenes — CHANGES_REQUESTED** [review]

File: flow/src/test/java/com/sequenceiq/flow/service/flowlog/FlowChainLogServiceTest.java (hunk 2 above)
Comment (kkalvagadda1) [reply flow/src/test/java/com/sequenceiq/flow/service/flowlog/FlowChainLogServiceTest.java]: fixed.

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**